Quickstart example not working

Hi,

I am unable to complete the quickstart tutorial.

My environment
Java 8
mvn: 3.6.0

I have cloned the entire project 
On step run "mvn verify" I receive the below error.  Please advise

mvn verify
[INFO] Scanning for projects...
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Build Order:
[INFO]
[INFO] quickstart                                                         [pom]
[INFO] rest                                                               [jar]
[INFO] app                                                                [jar]
[INFO]
[INFO] ----------< org.osgi.enroute.examples.quickstart:quickstart >-----------
[INFO] Building quickstart 0.0.2-SNAPSHOT                                 [1/3]
[INFO] --------------------------------[ pom ]---------------------------------
[INFO]
[INFO] -------------< org.osgi.enroute.examples.quickstart:rest >--------------
[INFO] Building rest 0.0.2-SNAPSHOT                                       [2/3]
[INFO] --------------------------------[ jar ]---------------------------------
[WARNING] The POM for org.osgi.enroute:osgi-api:pom:7.0.1-SNAPSHOT is missing, no dependency information available
[WARNING] The POM for org.osgi.enroute:enterprise-api:pom:7.0.1-SNAPSHOT is missing, no dependency information available
[WARNING] The POM for org.osgi.enroute:test-bundles:pom:7.0.1-SNAPSHOT is missing, no dependency information available
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for quickstart 0.0.2-SNAPSHOT:
[INFO]
[INFO] quickstart ......................................... SUCCESS [  0.026 s]
[INFO] rest ............................................... FAILURE [  0.184 s]
[INFO] app ................................................ SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  0.397 s
[INFO] Finished at: 2019-02-20T16:56:55-05:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project rest: Could not resolve dependencies for project org.osgi.enroute.examples.quickstart:rest:jar:0.0.2-SNAPSHOT: The following artifacts could not be resolved: org.osgi.enroute:osgi-api:pom:7.0.1-SNAPSHOT, org.osgi.enroute:enterprise-api:pom:7.0.1-SNAPSHOT, org.osgi.enroute:test-bundles:pom:7.0.1-SNAPSHOT: Failure to find org.osgi.enroute:osgi-api:pom:7.0.1-SNAPSHOT in https://oss.sonatype.org/content/groups/osgi was cached in the local repository, resolution will not be reattempted until the update interval of OSGi has elapsed or updates are forced -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :rest

--- a/examples/pom.xml
+++ b/examples/pom.xml
@@ -85,9 +85,20 @@
         <repository>
             <id>OSGi</id>
             <name>OSGi public binaries</name>
-            <url>https://oss.sonatype.org/content/groups/osgi</url>
+            <url>https://oss.sonatype.org/content/repositories/osgi-releases/</url>
             <releases>
                 <enabled>true</enabled>
+            </releases>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+        </repository>
+        <repository>
+            <id>OSGi Snapshots</id>
+            <name>OSGi public snapshot binaries</name>
+            <url>https://oss.sonatype.org/content/repositories/osgi-snapshots/</url>
+            <releases>
+                <enabled>false</enabled>
             </releases>
             <snapshots>
                 <enabled>true</enabled>

